modules/affinity/src/main/java/org/apache/ignite/internal/affinity/ditributed/AffinityManager.java

```diff
@@ -101,22 +95,22 @@ public AffinityManager(
                             UUID tblId = UUID.fromString(placeholderValue);
 
                             try {
-                                String name = new String(metaStorageMgr.service().get(
+                                String name = new String(metaStorageMgr.get(
```

Comment:
I'd rather retrieve table names and all other data that is supposed to be consistent from Vault. Otherwise carefully use get with upperBound revision.

Reply:
We decided to make it later, after the Vault interface would be prepared.